clients/web/src/templates/body/userList.pug

```diff
@@ -1,5 +1,10 @@
 .g-user-list-header
   .g-user-pagination
+  if getCurrentUser() && getCurrentUser().get('admin')
```

Comment:
Assuming that the template takes a `currentUser` parameter, you can [define a variable](https://pugjs.org/language/interpolation.html) at the top of the template:
```
- const isAdmin = currentUser && currentUser.get('admin');
```
Then all your `if` blocks can use it:
```
  if isAdmin
```